[IMP] localization/spain

toaa-odoo opened the pull request:
Updated the Spain localization page

taskid-3736761

LoredanaLrpz commented on content/applications/finance/fiscal_localizations/spain.rst:
```diff
@@ -2,34 +2,180 @@
+In order for **FACe** to work with **administrative centers**, the invoice *must* include specific
+data about the centers.
+
+.. tip::
+   Make sure to have the :guilabel:`Spain - Facturae EDI - Administrative Centers Patch
+   (l10n_es_edi_facturae_adm_centers)` module installed.
```
It's more of a necessity than just a tip. Also, I noticed you're consistently using the developer mode name (l10n...). Any particular reason for that?

toaa-odoo replied:
You're right, I'm changing it. I explained the reason in a comment a little further up :)

LoredanaLrpz commented on content/applications/finance/fiscal_localizations/spain.rst:
```diff
@@ -2,34 +2,180 @@
+
+To add **admnistrative centers**, create a new **contact** to add to the **partner** company. Select
+:guilabel:`FACe Center` as **type**, assign one or more **role(s)** to that contact, and
+:guilabel:`Save`. The **three** roles usually required are:
+
+- Órgano gestor: :guilabel:`Receptor` (Receiver);
+- Unidad tramitadora: :guilabel:`Pagador` (Payer);
+- Oficina contable: :guilabel:`Fiscal` (Fiscal).
```
Why these roles? Can we have a bit of explanation?

toaa-odoo replied:
The info in this doc comes straight from Spanish BAs in Valencia and is the document they use, it's not included in it and goes beyond my understanding. I believe users dealing with their accounting would know what the roles correspond to. IMO I do not believe it is necessary to explain them

LoredanaLrpz commented on content/applications/finance/fiscal_localizations/spain.rst:
```diff
@@ -2,34 +2,180 @@
+To select the one to use, go to :menuselection:`Accounting --> Configuration --> Settings`, and
+select a package in the :guilabel:`Fiscal localization` section.
 
 .. tip::
     When you create a new Odoo Online database, the PGCE PYMEs 2008 is installed by default.
 
```
PGCE PYME --> English version

LoredanaLrpz commented on content/applications/finance/fiscal_localizations/spain.rst:
```diff
@@ -2,34 +2,180 @@
+- Tax Report (Modelo 349);
+- Tax Report (Modelo 390).
+
+You can access Spanish-specific versions of reports by clicking on the **book** icon when on a
+report and selecting its Spanish version: :guilabel:`(ES)`.
```
There's only "one" version of every report, I'd say "You can switch between Spanish-specific tax reports by clicking..."

LoredanaLrpz commented on content/applications/finance/fiscal_localizations/spain.rst:
```diff
@@ -2,34 +2,180 @@
+Once a region selected, click :menuselection:`Manage certificates (SII/TicketBAI) --> New`, upload
+the certificate provided by the tax agency and enter the password also provided by the agency.
+
+.. image:: spain/certificate.png
+   :alt: Password and validity of tax agency certificate.
+
+.. warning::
+   If you are testing certificates, make sure to enable :guilabel:`Test Mode` under
+   :menuselection:`Settings --> Accounting --> Spain Localization (Section)`.
```
There was also a note here: https://runbot131.odoo.com/runbot/static/build/57980674-17-0/logs/build/html/applications/finance/fiscal_localizations/spain.html; they both disappeared, but I don't understand why? The files provided were useful for testing the flow.

toaa-odoo replied:
These links were meant for internal use only

LoredanaLrpz commented on content/applications/finance/fiscal_localizations/spain.rst:
```diff
@@ -2,34 +2,180 @@
+--------
+
+Once you have :doc:`created <../../finance/accounting/customer_invoices>` an invoice and confirmed
+it, click :guilabel:`Send and Print`. Make sure :guilabel:`Generate Facturae edi file` is enabled,
```
```suggestion
it, click :guilabel:`Send & Print`. Make sure :guilabel:`Generate Facturae edi file` is enabled,
```

LoredanaLrpz reviewed (changes requested):
Hey @toaa-odoo , thanks for being quick to answer the BA's need for documentation. I have a few comments and suggestions, I'll make myself available next week if anything isn't clear.
Mainly:
- I've checked the task, but I could find any specs: where are all these information coming from?
- Could you provide more explanation for each concept? Like an introductory line just to understand what you're talking about. 
- I've also noticed a lot of "make sure" when there's no need. 
- I don't think you should write terms in Spanish if they are written in English on the DB, I think they'll read the Spanish doc if they want Spanish.

I'm looking forward to read the reviewed version of your doc :)

Thanks again !

toaa-odoo commented on content/applications/finance/fiscal_localizations/spain.rst:
```diff
@@ -2,34 +2,180 @@
+Taxes
+=====
+
+Default Spanish taxes are created automatically when the
+:guilabel:`Spanish - Accounting (PGCE 2008)`, and the :guilabel:`Spanish - Accounting (PGCE 2008)`
```
Modules is the official name of extensions/addons/packages in Odoo when looking for apps, in my screenshot you can see the search bar says "module", and some other extensions say "Module info"
![modules](https://github.com/odoo/documentation/assets/104567387/1d3bec16-5a73-4735-897f-f07596dd3197)

toaa-odoo commented on content/applications/finance/fiscal_localizations/spain.rst:
```diff
@@ -2,34 +2,180 @@
+
+Default Spanish taxes are created automatically when the
+:guilabel:`Spanish - Accounting (PGCE 2008)`, and the :guilabel:`Spanish - Accounting (PGCE 2008)`
+modules are installed.  Each tax impacts the Spanish **Tax Reports (Modelo)**, available by going to
```
"Mod" is short for Modelo, which the is the name of that tax report in Spain (there's no translation, it's a localization thing), I put it in bold and not guilabel for that reason (I'll change to Spain-specific tax reports (modelo) for clarity

toaa-odoo commented on content/applications/finance/fiscal_localizations/spain.rst:
```diff
@@ -2,34 +2,180 @@
+- Balance Sheet;
+- Profit & Loss;
+- EC Sales List;
+- Tax Report (Modelo 111);
+- Tax Report (Modelo 115);
+- Tax Report (Modelo 303);
+- Tax Report (Modelo 347);
+- Tax Report (Modelo 349);
+- Tax Report (Modelo 390).
```
Same as above

toaa-odoo commented on content/applications/finance/fiscal_localizations/spain.rst:
```diff
@@ -2,34 +2,180 @@
+-------------
+
+Odoo supports the **TicketBAI (TBAI)** electronic invoicing format for all three regions of the
+**Basque Country (Álava, Biscay, Gipuzkoa)**. To enable **TicketBAI**, make sure to first set the
```
Biskaia is the Spanish spelling, Biscay is the English spelling. Because the page is in English I wrote in English

toaa-odoo commented on content/applications/finance/fiscal_localizations/spain.rst:
```diff
@@ -2,34 +2,180 @@
+General Settings --> My Company` in the :guilabel:`Companies` section, and clicking
+:guilabel:`Update Info`.
+
+Then, make sure to have the module :guilabel:`Spain -TicketBAI (l10n_es_edi_TBAI)` installed. If not
```
l10n_es_edi_TBAI is the technical name of the module. You can see it when switching into developer mode, I forgot to to do that for this screenshot. The reason for including both the technical name *and* the module's name is if in the future the module's name changes, the technical name will always be the same and therefore remains identifiable

toaa-odoo commented on content/applications/finance/fiscal_localizations/spain.rst:
```diff
@@ -2,34 +2,180 @@
+Use case
+--------
+
+Once an invoice has been :doc:`created <../../finance/accounting/customer_invoices>` and confirmed,
+a TicketBAI **banner** appears at the top. Odoo sends invoices through TicketBAI automatically every
+**24 hours**. However, you can click :guilabel:`Process now` to send the invoice immediately.
```
You need specific test credentials which I had access to, don't worry too much about this

toaa-odoo commented on content/applications/finance/fiscal_localizations/spain.rst:
```diff
@@ -2,34 +2,180 @@
+Taxes
+=====
+
+Default Spanish taxes are created automatically when the
```
The taxes shown in the screenshot are just the regular purchase and sales tax, which are also the most used ones, hence why they are in the Settings apps for quick access and setup. Localization packages also come with other taxes (not visible here) that are accessible through Accounting --> Configuration --> Accounting: Taxes and relate to other things, such as assets-specific taxes, HR-specific taxes, etc all complying with the Spain accounting. We are trying to establish a common layout to all localizations, with some having long specific taxes section, and others not at all. In this case the section is just there to follow that layout and specify/remind that the taxes installed are compliant with Spain Accounting

toaa-odoo commented on content/applications/finance/fiscal_localizations/spain.rst:
```diff
@@ -2,34 +2,180 @@
+
+.. warning::
+   If you are testing certificates, make sure to enable :guilabel:`Test Mode` under
+   :menuselection:`Settings --> Accounting --> Spain Localization (Section)`.
```
Because you are already in "Settings" to configure TicketBAI

toaa-odoo commented on content/applications/finance/fiscal_localizations/spain.rst:
```diff
@@ -2,34 +2,180 @@
+you can see the traceability of other generated documents related to the invoice (ex. if the invoice
+should also be sent through the **SII**, it will appear here).
+
+.. image:: spain/chatter.png
+   :alt: XML file in the chatter of the invoice.
```
Localizations are usually only done once during the configuration of a new DB or company. A new DB or company implies possibly a new user, and therefore they would not be familiar with Odoo's interface and terms. I believe the screenshot helps identify what a chatter is and what the file would look like.

toaa-odoo commented on content/applications/finance/fiscal_localizations/spain.rst:
```diff
@@ -2,34 +2,180 @@
+
+Administrative centers
+----------------------
+
```
See the comment below about roles

toaa-odoo commented:
@LoredanaLrpz I've added the links to the Spanish Knowledge articles which contain the info used for this doc (specs), however, I'm not sure you have access. They may have been granted to me specifically for this doc.

LoredanaLrpz reviewed (approved):
hey @toaa-odoo , I still have three comments but I'm already approving it so you can ping doc-review after making the changes =)

auva-odoo reviewed (changes requested):
Hello @toaa-odoo thank you for your work !
- General question regarding e-Invoicing: you explain how the XML file is available in the chatter, but can you also download it like you can for Peppol for example ? (I was not able to test this myself, hence the question) If that's the case, I'm not sure we should talk about the chatter so much (and defintely remove the screesnhots, they are not necessary IMO)
- I suggested to remove most screenshots (not all of them, see my individual comments) because I don't think they add much value for the weight + maintenance they will require. We can discuss it if you want ofc
- Plus a few suggestions/questions
Let me know if you want to discuss any of these (might be easier to show me some things). Thanks again!

toaa-odoo commented on content/applications/finance/fiscal_localizations/spain.rst:
```diff
@@ -2,34 +2,187 @@
+
+.. warning::
+   If you are testing certificates, make sure to enable :guilabel:`Test Mode` under
+   :menuselection:`Settings --> Accounting` in the :guilabel:`Spain Localization` section.
```
They would already be in the "Settings" app when setting up TickatBAI, making them leave through the accounting app only to end up in the "Settings" app again seems counter-productive to me

auva-odoo replied:
You're right. Make sure to rephrase the sentence a little bit then, since Accounting is not a menu item.

toaa-odoo commented:
@auva-odoo Hi Audrey! I addressed your comments above. As for PEPPOL, any EDI system (X-Factur, E-FFF, etc.) always appears in the chatter either as XML or PDF depending on the system :)

auva-odoo reviewed (approved):
@robodoo r+
